packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1701,6 +1713,12 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
             return;
         }
 
+        this.mc.logger.sendTelemetryEvent({
```

Comment:
I think today this API is called by pending state manager, and thus this new event will fire all the time, creating a lot of noise and not letting us know anything new.

Reply:
Not sure if that's the case. Indeed, it is called, but if it does not change, nothing will be logged. Everything else seems to call this in order to preserve the existing flushmode and ensure it is reverted after a series of ops which may mutate it. If it does not change, there will be no noise.

Reply:
Verified with some examples, this should not fire unless the flushmode is changed explicitly. Also, https://github.com/microsoft/FluidFramework/issues/9398 proves the API is not actually used..